🐛 Install command failed

### Prerequisites

- [X] read documentation
- [X] searched open issues for existing bug report
- [X] searched closed issues for solution(s) or feedback

### Current behavior

```
npm i @flex-development/docast-parse@flex-development/docast-parse
npm ERR! code 1
npm ERR! command failed
npm ERR! command sh -c chmod +x .husky/* && husky install
npm ERR! chmod: cannot access '.husky/*': No such file or directory
```

### Reproduction

```
npm i @flex-development/docast-parse@flex-development/docast-parse
```

### Expected behavior

It should be installed

### Package version

-

### Node.js version

v18.12.1

### Package manager version

9.2.0

### Operating system (environment)

linux

### Additional context

Hello.

I want to try your package, but the install command throw the error.

--- a/package.json
+++ b/package.json
@@ -60,7 +60,7 @@
     "fix:dedupe": "yarn dedupe --strategy=highest",
     "fix:format": "prettier --cache --write .",
     "fix:lint": "yarn check:lint --cache --fix",
-    "postinstall": "chmod +x .husky/* && husky install",
+    "postinstall": "[ -f ./node_modules/.bin/husky ] && chmod +x .husky/* && husky install || exit 0",
     "postpack": "toggle-scripts +postinstall",
     "postpublish": "toggle-scripts +prepack",
     "prepack": "toggle-scripts -postinstall && yarn build",

